cypress/e2e/7-update-profile.js

```diff
@@ -0,0 +1,43 @@
+describe(
+  'Colony user can update their profile',
+  () => {
+    const userSettingsName = 'ColonyFan';
+    const userSettingsBio = 'I really like Colony';
+    const userSettingsWebsite = 'http://colony.io';
+    const userSettingsLocation = 'Earth';
+    const profilePicturePath = 'src/img/favicon.png';
+
+    beforeEach(() => {
+      cy.login();
+      cy.visit(`/landing`);
```

Comment:
I think you don't need this here as `.login()` already visits `/landing`

Reply:
That is good to know, thank you, updated!